src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,25 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MainApp {
+    public static void main(String[] args) {
+
+        Truck truck = new Truck("Truck");
+        Bulldozer bulldozer = new Bulldozer("Bulldozer");
+        Excavator excavator = new Excavator(" Excavator");
+
+
+        Machine[] machines = {truck, bulldozer, excavator};
+        printTotal(machines);
+        System.out.println();
+    }
 
+    private static void printTotal(Machine[] machines) {
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+        System.out.println(Arrays.toString(machines));
```

Comment:
Printing the array directly using `Arrays.toString(machines)` will not provide informative messages about the types of machines. It's better to override the `toString` method in each machine class to provide a more informative message or to iterate through the array and print out details of each machine individually.